Week3/homework/step2-3.js

```diff
@@ -31,8 +33,9 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  do {
```

Comment:
Because the `do-while` loop test the loop condition at the end of the loop (and therefore executes the loop body at least once), you must test yourself, before entering the loop, whether `num <=0`.